app/src/main/java/com/kylecorry/trail_sense/astronomy/infrastructure/commands/SunsetAlarmCommand.kt

```diff
@@ -20,20 +20,20 @@ import java.time.ZonedDateTime
 
 class SunsetAlarmCommand(private val context: Context) : CoroutineCommand {
 
-    private val location by lazy { LocationSubsystem.getInstance(context) }
-    private val userPrefs by lazy { UserPreferences(context) }
+    private val location: LocationSubsystem = LocationSubsystem.getInstance(context)
+    private val userPrefs: UserPreferences = UserPreferences(context)
     private val astronomyService = AstronomyService()
 
-    // The window prior the alert time that the alert can be sent
-    private val alertWindow = Duration.ofMinutes(20)
+    private val alertWindowMinutes = 20
```

Comment:
Can you restore this alertWindow line and the comment - I prefer this to be inlined rather than two separate values